index.html

```diff
@@ -24,13 +24,19 @@
         <iframe
           width="900"
           height="506"
-          src="https://www.youtube.com/embed/EqQCfC7KHlQ"
-          title="I Died For You"
+          src="https://www.youtube.com/embed/5vz4yIgnFt0"
+          title="ICED EARTH - Anthem (OFFICIAL VIDEO)"
           frameborder="0"
           allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
           allowfullscreen
         ></iframe>
       </div>
     </div>
+    <br />
```

Comment:
Please style these divs so that they all have equal height as the horizontal ones. 
Also  make the class vertical with width of 33% and position it in the center of the body element (hint you can style the body element or you can wrap everything in your own div and give it class name and style that div)

Reply:
Updated as suggested